statspost/client.py

```diff
@@ -36,6 +36,7 @@
     "infinity",
     "motiondevelopment",
     "discordservices",
+    "TopicList",
```

Comment:
please change this to 
```python
"topiclist",
```

Reply:
see due to this these errors are happening https://deepsource.io/gh/Dhruvacube/statspost.py/run/1e89b256-2dd9-4b26-8dda-645d1a1c4542/python/FLK-E114

Reply:
https://deepsource.io/gh/Dhruvacube/statspost.py/run/1e89b256-2dd9-4b26-8dda-645d1a1c4542/python/FLK-E265